op-node/eth/label.go

```diff
@@ -17,3 +17,9 @@ const (
 	// - L2: Derived chain tip from finalized L1 data
 	Finalized = "finalized"
 )
+
```

Comment:
compile time interface check for `rpcBlockID`?

Reply:
It's a bit verbose for a private interface to export it just for a compile time check, it'd be a circular dependency (or have to move the interface), and the sources package will already complain if a block label cannot be used as `rpcBlockID`.